apps/pages/urls.py

```diff
@@ -3,6 +3,34 @@
 from . import views
 
 urlpatterns = [
+    #Gerenal Pages
     path('', TemplateView.as_view(template_name="page/Home.html")),
     path('about/', TemplateView.as_view(template_name="page/About.html")),
```

Comment:
@dhmm Check [my PR,](https://github.com/chadoulis/forecasts/pull/3) we will have conflict here.